source/framefmt.c

```diff
@@ -203,6 +203,12 @@ static inline int serde_nonframed(
     struct aws_cryptosdk_framestate *AWS_RESTRICT state, struct aws_cryptosdk_frame *AWS_RESTRICT frame) {
     field_sized(state, &frame->iv, state->alg_props->iv_len);
     field_be64(state, &state->plaintext_size);
+
+    // Check that plaintext_size is within bounds so we avoid an overflow later
```

Comment:
comments on C style.
```c
/* Check that plaintext_size is within bounds so we avoid an overflow later */
```

Reply:
Good catches on the comment style, I thought his // comments were only in one file, I'll touch that up.

Reply:
Reformat.sh leaves these style of in-function comments untouched, so it seems like the preferred comment style for this project.

Reply:
Right!! The important thing is just to follow the `reformat.sh`.